boot/freeldr/freeldr/include/arch/pc/x86common.h

```diff
@@ -21,6 +21,7 @@
 #define BIOSCALLBUFSEGMENT (BIOSCALLBUFFER/16) /* Buffer to store temporary data for any Int386() call */
 #define BIOSCALLBUFOFFSET   HEX(0000) /* Buffer to store temporary data for any Int386() call */
 #define BIOSCALLBUFSIZE     PAGE_SIZE /* max is sizeof(VESA_SVGA_INFO) = 512 */
+#define MAX_BIOS_DESCRIPTORS 80ul
```

Comment:
ok so can we remove that "ul" thingie?

Reply:
Alright, let it be so, but I still didn't figured out why it was defined like that.